Changelog.md

```diff
@@ -1,5 +1,16 @@
 ## Gazebo Math 6.x
 
+## Gazebo Math 6.13.0 (2022-08-31)
+
+1. Support migration
+    * [Pull request #483](https://github.com/gazebosim/gz-math/pull/483)
+
+1. Migrate headers
+    * [Pull request #483](https://github.com/gazebosim/gz-math/pull/483)
```

Comment:
I would merge these two entries comprising #483

Reply:
Merged! And I shall merge :eyes: